jstests/libs/backup_utils.js

```diff
@@ -128,7 +128,9 @@ function _copyFileHelper(absoluteFilePath, sourceDbPath, destinationDirectory) {
     // Grab the file path relative to the dbpath. Maintain that relation when copying
     // to the `hiddenDbpath`.
     let relativePath = absoluteFilePath.substr(sourceDbPath.length);
-    let destination = destinationDirectory + '/' + relativePath;
+    let destination = destinationDirectory + separator + relativePath;
+    const newFileDirectory = destination.substring(0, destination.lastIndexOf(separator));
```

Comment:
If the destination directory already exists, then the `new` part in the `newFileDiretory` may sound confusing. Do you think `destinationDir` name is a better name?